diff -urN libsocialweb-0.25.20/configure.ac libsocialweb-0.25.20-patched/configure.ac
--- libsocialweb-0.25.20/configure.ac	2011-11-12 21:56:37.000000000 +1100
+++ libsocialweb-0.25.20-patched/configure.ac	2012-09-01 01:28:59.768127927 +1100
@@ -62,7 +62,7 @@
 
 GOBJECT_INTROSPECTION_CHECK([0.9.6])
 
-PKG_CHECK_MODULES(GLIB, glib-2.0 >= 2.14)
+PKG_CHECK_MODULES(GLIB, gmodule-2.0 glib-2.0 >= 2.14)
 PKG_CHECK_MODULES(GIO, gio-2.0)
 PKG_CHECK_MODULES(GOBJECT, gobject-2.0 >= 2.14)
 PKG_CHECK_MODULES(GCONF, gconf-2.0)
diff -urN libsocialweb-0.25.20/libsocialweb/Makefile.am libsocialweb-0.25.20-patched/libsocialweb/Makefile.am
--- libsocialweb-0.25.20/libsocialweb/Makefile.am	2011-08-10 20:38:28.000000000 +1100
+++ libsocialweb-0.25.20-patched/libsocialweb/Makefile.am	2012-09-01 01:39:49.518128008 +1100
@@ -8,12 +8,12 @@
 libsocialweb_ladir = $(pkgincludedir)/libsocialweb
 
 libsocialweb_la_CFLAGS = -I$(top_srcdir) -I$(top_srcdir)/interfaces \
-		     $(DBUS_GLIB_CFLAGS) $(SOUP_CFLAGS) $(SOUP_GNOME_CFLAGS) \
+		     $(GLIB_CFLAGS) $(DBUS_GLIB_CFLAGS) $(SOUP_CFLAGS) $(SOUP_GNOME_CFLAGS) \
 		     $(NM_CFLAGS) $(GTK_CFLAGS) $(REST_CFLAGS) \
 		     $(GCOV_CFLAGS) \
 		     -DSOCIALWEB_SERVICES_MODULES_DIR=\"$(servicesdir)\"
 
-libsocialweb_la_LIBADD = $(DBUS_GLIB_LIBS) $(SOUP_LIBS) $(SOUP_GNOME_LIBS) \
+libsocialweb_la_LIBADD = $(GLIB_LIBS) $(DBUS_GLIB_LIBS) $(SOUP_LIBS) $(SOUP_GNOME_LIBS) \
 		      $(NM_LIBS) $(GTK_LIBS) $(REST_LIBS) \
 		      $(GCOV_LDFLAGS) \
 		      $(top_builddir)/interfaces/libsocialweb-ginterfaces.la
